test/DataContainers/runtests.jl

```diff
@@ -39,4 +39,11 @@ using EnsembleKalmanProcesses.DataContainers
     @test get_inputs(iopairs) == parameter_samples
     @test get_outputs(iopairs) == data_samples
 
+    # test deep-copying
+    parameter_samples[1, 1] += 1
+    @test !isequal(get_data(idata), parameter_samples)
+    retrieved_samples = get_data(idata)
+    retrieved_samples[1, 1] += 1
+    @test !isequal(retrieved_samples, get_data(idata))
```

Comment:
Could you add similar tests for `idata_T`?

Reply:
done